sample-functions/AlpineFunction/Dockerfile.arm64

```diff
@@ -1,8 +1,8 @@
 FROM arm64v8/alpine:3.7
```

Comment:
FYI we can remove the prefix now for `arm64v8` because Docker released an official multi-arch image under the tag `alpine:3.7`

Reply:
Nice, I'll update that as well.